### PR TITLE
Using Global CSS root variables across all files

### LINKED ISSUE #2983: Using Global CSS root variables for better consistency
**Is your feature request related to a problem? Please describe.**
The hardcoded values in the tsx files for the styles such as those present in the 'sx' prop can be replaced with global css root variables. 

**Describe the solution you'd like**
Simply import the styles from the global CSS and therefore use the global root variables.

For example :

For this style
original

We have the output:
2nd

But by simply replacing the hardcoded backgroundcolor value with global variable root variable we can ensure much higher consistency for the CSS:
after change

We get the same result:
4th

**Approach to be followed (optional)**
1. Visit all the .tsx files in the `src` directory.
2. Ensure they have styles imported from global css something like this: `import styles from '../../style/app.module.css'; `
3. Replace all the possible hard coded values with global css root variables.
4. Run the server before making the changes and after the changes making sure CSS do not gets disrupted for any component.

**Potential internship candidates**

Please read this if you are planning to apply for a Palisadoes Foundation internship 
- https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by AceHunterr**
**What kind of change does this PR introduce?**
Feature, more like providing consistency

**Issue Number:**
Fixes #2983 

**Did you add tests for your changes?**
No

**Snapshots/Videos:**
The files whose styles are moved to global css module i.e - `src/style/app.module.css` were looked upon and if those files had common standard css styling needed they were replaced by the root css vars. Currently only these files have been refactored with global css yet so these were looked upon:

**Summary**

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

## Summary by CodeRabbit

- **Style Updates**
	- Replaced hardcoded color values with CSS variables across multiple components for improved styling consistency.
	- Added new CSS variable `--primary-border-solid` for standardized border styling.
	- Updated border and background colors in components like AddOnEntry, EventAttendance, VolunteerGroupViewModal, ManageTag, and OrganizationTags.

- **Code Cleanup**
	- Removed commented-out code in Requests and AddOnEntry components.
	- Simplified import statements in EventRegistrants component.

- **Minor Improvements**
	- Enhanced error handling in OrgList component.
	- Refined organization search and loading functionality.

### CODE REVIEW

**Comment by AceHunterr** [comment]
@palisadoes Since we are still in the transition of making a global css file for all I see many repeated css style which are not used as variable for example:

I did a global search and found this style being used 7-8 places in the code
`border : 1px solid #31bb6b`
So I basically created `  --primary-border-green: 1px solid #31bb6b;` as a root variable in the CSS file

But the issue lies that out of these 7-8 files only 1 was refactored with our single global css file. So unless all the other files are not refactored we won't be able to use the style inside them. 

Although it won't be much time consuming issue once all files are refactored... if I would change them right now it would cause a lot of merge conflicts to the already assigned assignees. So I would request that you may merge this PR upon reviewing but the issue will have to be open for sometime I guess.

File: src/style/app.module.css [hunk 1]
```diff
@@ -31,6 +31,7 @@
   --tablerow-bg-color: #eff1f7;
   --row-background: var(--bs-white, white);
   --font-size-header: 16px;
+  --primary-border-green: 1px solid #31bb6b;
```
Comment (palisadoes) [inline src/style/app.module.css:34]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466 
1. This needs to be changed to `--primary-border` or something non color specific or use an existing border color. We are moving away from green. There are other border colors

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

File: src/style/app.module.css (hunk 1 above)
Comment (AceHunterr) [reply src/style/app.module.css:34]: Okay sure I'll look into that

**Comment by AVtheking** [comment]
Please fix the conficting files.

**Comment by AceHunterr** [comment]
> Please fix the conficting files.

Done

**Comment by palisadoes** [comment]
Please address the green coloration I mentioned previously.

**Review by palisadoes — APPROVED** [review]

**Comment by AceHunterr** [comment]
> Please address the green coloration I mentioned previously.

I was looking for the alternate color in the PR mentioned

  `--dropdown-border-color: #555555;`
  
  then I used this color for the alternative as most of the borders were using this in the code... is it good to go?

**Comment by palisadoes** [comment]
That should work. It's sufficiently dark

**Pull request merged**